lib/display.rb

```diff
@@ -11,10 +11,13 @@ def display_board(board)
   def ask_for_move
     print 'Choose a position from 1-9: '
     move = $stdin.gets.chomp
-    move.to_i
   end
 
   def show_current_player(current_player_mark)
     print "The current player is #{current_player_mark}\n"
   end
+
+  def invalid_move_message(move)
```

Comment:
I like how you used `show` verb in the other method (show_current_player), it was more descriptive (here the method doesn't say it prints stuff.

Reply:
Thank you, I'll change the method names